.github/workflows/experimental.yml

```diff
@@ -54,13 +54,55 @@ jobs:
       DOCKER_IMAGE_TAG: ${{ matrix.version }}
       DOCKER_IMAGE_WEBLOG_TAG: ${{ github.sha }}
       BUILDX_PLATFORMS: linux/amd64
+      MODE: manual
     steps:
+      - name: lib-injection test runner
+        id: lib-injection-test-runner
+        # TODO: change this back to `main`
+        uses: DataDog/system-tests/lib-injection/runner@onboarding-rc-injection
+        with:
+          docker-registry: ghcr.io
+          docker-registry-username: ${{ github.actor }}
+          docker-registry-password: ${{ secrets.GITHUB_TOKEN }}
+          test-script: ./lib-injection/run-manual-lib-injection.sh
 
+  auto-lib-injection-tests:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    strategy:
+      matrix:
+        variant:
+        - library: java
+          weblog-variant: dd-lib-java-init-test-app
+        - library: nodejs
+          weblog-variant: sample-app
+        - library: python
+          weblog-variant: dd-lib-python-init-test-django
+        test-case: 
+          - TC1
```

Comment:
I'm not a big fan of these names, I suppose they could be changed to something more descriptive or if not, document these names in the code or in the readme.md

Reply:
Good point, what about `TestCaseN`?

Reply:
FWIW each test case is documented step by step in `run-auto-lib-injection.sh`